apps/store/src/components/ProductPage/PurchaseForm/DeductibleSelector.tsx

```diff
@@ -45,14 +45,7 @@ export const DeductibleSelector = ({ offers, selectedOffer, onValueChange }: Pro
       }
     })
 
-    // Sort deductibles based on monthly price
-    const sortedLevels = levels.sort((a, b) => {
-      if (a.price.amount < b.price.amount) return -1
-      if (a.price.amount > b.price.amount) return 1
-      return 0
-    })
-
-    return sortedLevels
+    return levels
```

Comment:
Don't we need to call `getOfferByPrice` here? 🤔 
```suggestion
    return getOfferByPrice(levels)
```

Reply:
What function is that? Can't find it